src/serlo-editor/plugins/fill-in-the-gap-exercise/components/gap-solution-area.tsx

```diff
@@ -0,0 +1,17 @@
+// import { useDroppable } from '@dnd-kit/core'
```

Comment:
This is commented out code from my experiments getting drag & drop to work. I chose to leave it in because I'm going to built upon that in the next iteration. But it definitely makes the code more messy right now.